src/pages/ResultsListPage.js

```diff
@@ -101,8 +101,11 @@ class ResultsListPage extends BaseResultsPage {
     }
 
     renderPersonalisationLink(): ReactElement {
+        const current = this.props.location.pathname;
+        const separator = current.endsWith("/") ? "" : "/";
```

Comment:
ergh, relative URLs lose the last segment of the path. Don't think that will work.